BourbonAndBurial/Data/OrderRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Dapper;
+using System.Threading.Tasks;
+using BourbonAndBurial.Models;
+using System.Data.SqlClient;
+
+namespace BourbonAndBurial.Data
+{
+    public class OrderRepository
+    {
+        const string ConnectionString = "Server=localhost;Database=BourbonAndBurial;Trusted_Connection=True;";
+
+
+        public CreateOrderRequest AddOrder(int customerId, int paymentTypeId)
+        {
+
+            //const string ConnectionString = "Server=localhost;Database=BourbonAndBurial;Trusted_Connection=True;";
+
+            using (var db = new SqlConnection(ConnectionString))
+            {
+                var insertQuery = @"
+                        INSERT INTO [dbo].[Order]
+                                   ([CustomerId]
+                                   ,[PaymentTypeId])
+                        output inserted.*
+                             VALUES
+                                   (@customerId,
+                                    @paymentTypeId";
```

Comment:
I was getting an error when trying to add an order because there isnt a closing parenthesis. 
line 30 should be ```@paymentTypeId)";```